Assets/Scripts/MIDI2Event/MTrkTypes/SetTempoMeta.cs

```diff
@@ -1,4 +1,4 @@
-namespace midi2event
+namespace MIDI2EventSystem
 {
     internal class SetTempoMeta : MTrkEvent
```

Comment:
add comment saying what US and MS stands for (something tells me that MS does not mean Multiple Sclerosis in this case, but Google's search seems to think otherwise)

Reply:
US is standard abbreviation for microseconds, added comment just in case. Changed MS to US.